camera.go

```diff
@@ -293,8 +319,36 @@ type MouseZoomer struct {
 func (*MouseZoomer) Priority() int          { return 10 }
 func (*MouseZoomer) Remove(ecs.BasicEntity) {}
 
-func (c *MouseZoomer) Update(dt float32) {
+func (c *MouseZoomer) Update(float32) {
 	if Mouse.ScrollY != 0 {
+		// TODO: not sure if we have to use dt here, since we already use the "amount" of zooming
 		Mailbox.Dispatch(CameraMessage{Axis: ZAxis, Value: Mouse.ScrollY * c.ZoomSpeed, Incremental: true})
 	}
 }
+
+// MouseRotator is a System that allows for zooming when the scroll wheel is used
+type MouseRotator struct {
+	RotationSpeed float32
+
+	oldX    float32
+	pressed bool
+}
+
+func (*MouseRotator) Priority() int          { return 10 }
```

Comment:
@paked The only change that would make sense, is change it to `float32` (to allow decimals).